README.md

```diff
@@ -1,36 +1,54 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
-
 ## Getting Started
 
-First, run the development server:
+### Step 1: Install MongoDB
 
-```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-# or
-bun dev
-```
+1. Go to the [MongoDB Download Center](https://www.mongodb.com/try/download/community).
+2. Select your operating system and download the installer.
+3. Follow the installation instructions for your OS.
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+### Step 2: Install MongoDB Compass (GUI)
 
-You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
+1. Download MongoDB Compass from the [MongoDB Compass Download Page](https://www.mongodb.com/try/download/compass).
+2. Install Compass by following the provided instructions.
 
-This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
+### Step 3: Connect to MongoDB using Compass
 
-## Learn More
+1. Open MongoDB Compass.
+2. In the connection window, enter your connection string:
+   - For a local instance, use: `mongodb://localhost:27017`
+   - For a remote instance, use your provided connection string (e.g., `mongodb+srv://<username>:<password>@cluster.mongodb.net/<dbname>`).
+3. Click "Connect" to establish the connection.
+4. Once connected, you can create a new database by clicking on "Create Database" and entering the database name and collection name.
 
-To learn more about Next.js, take a look at the following resources:
+### Step 4: Install Required Packages
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+```bash
+npm install
+# or
+yarn install
+# or
+pnpm install
+# or
+bun install
+```
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+You will need to set up the following environment variables in your `.env` file for MongoDB:
 
-## Deploy on Vercel
+```
+MONGO_USER= sharedDevReader
```

Comment:
Redact this, but edit the .env.example if it is missing any creds that are needed. Maybe just leave a note to rename .env.example to .env.local.